website_docs/exporting_data.md

```diff
@@ -1,6 +1,7 @@
 ---
-title: "Processing and Exporting Data"
+title: Processing and Exporting Data
 weight: 4
+linkTitle: Processing and Exporting data
```

Comment:
--> `linkTitle: Exporting data` link it was in the previous edit. Thanks

Reply:
I made the change since the document is currently about more than just exporting data. Happy to revert, but I felt that this made more sense for the time being.

Reply:
The rational for the shorter titles is that space is at a premium in the left sidenav. So a strategy is to keep the full title for the page, but abbreviate it in the (now somewhat heavily indented) left sidenav.

I'll let it up to you, but if you keep both the same, then drop the `linkTitle` entry, it's unnecessary when the two are identical.

Reply:
Or, it could be just "Data".

Reply:
Gotcha, hadn't considered that. I'm cool with that change (and likely splitting this doc up later on, so there's distinct docs for sampling, processing, and exporting)